frontend/src/component/sidebar/info.vue

```diff
@@ -91,17 +98,118 @@ export default {
   data() {
     return {
       actions: [],
+      map: null,
+      marker: null,
+      mapLoaded: false,
+      loadingMapLibre: false,
     };
   },
   computed: {
     model() {
       return this.modelValue;
     },
+    megapixels() {
+      if (!this.model.Width || !this.model.Height) return 0;
+      return ((this.model.Width * this.model.Height) / 1000000).toFixed(1);
+    },
+  },
+  watch: {
+    "model.Lat"() {
+      this.loadMapAndInit();
+    },
+    "model.Lng"() {
+      this.loadMapAndInit();
+    },
+  },
+  mounted() {
+    if (this.model.Lat && this.model.Lng) {
```

Comment:
There might be a more elegant way than adding two model watchers and calling `loadMapAndInit()` (up to) 3 times? This looks like it could use refactoring :)

Reply:
For example, you could create a reusable component for the map (which is only rendered when there are coordinates) and load MapsLibreGL there in `mounted()`.